src/app/components/cds/media-cards/MediaCount.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import ButtonMain from '../../cds/buttons-checkboxes-chips/ButtonMain';
+import Tooltip from '../../cds/alerts-and-prompts/Tooltip';
+import MediaIcon from '../../../icons/perm_media.svg';
+import MediaTypeDisplayIcon from '../../media/MediaTypeDisplayIcon';
+import { getCompactNumber } from '../../../helpers';
+
+const MediaCount = ({
+  intl,
+  mediaCount,
+  mediaType,
+}) => (
+  <FormattedMessage
+    id="sharedItemCard.medias"
+    // {mediaCount, number} tells react-intl to format the number as Intl.NumberFormat(locale, {})
```

Comment:
@amoedoamorim Thanks for the review. I totally changed how the labels work here so instead of a separately localized number and a separately pluralized string, we are using the internal react-intl NumberFormat API and doing a fully localized string+number together, embedded in the `plural` directive.